### ISSUE TITLE
release ?

### ISSUE THREAD

**Issue by vtorri** [Samsung/rlottie#159]
hello

do you think that you will proveide a release tarball soon ?

thank you

**Comment by smohantty** [comment]
@vtorri ,
You mean source code release or debian package (.deb file) ?
Till now most users are building from the source.

**Comment by hermet** [comment]
@vtorri we haven't any plan yet since it's under intensive development still.
@smohantty it sounds @vtorri needs tarball (source code to build and install) not debian, guess https://mesonbuild.com/Creating-releases.html is the way to go.

**Comment by hermet** [comment]
@smohantty  Beaware this, once we tag versioning, we can't change any interfaces nor behavior, we need to discuss a schedule for official versioning.

**Comment by hermet** [comment]
@vtorri plus, rlottie won't be contained in efl projects. This is an independent project, efl might needs to install rlottie package if it needs to use lottie feature. Just discussed the release of rlottie with @smohantty, we are looking at the end of this year for a official v1.0 release of rlottie. After then, we will try to push this to debian or some other package distributors.

**Comment by vtorri** [comment]
@hermet, yes, i need a source tarball. I want to add it to ewpi (https://github.com/vtorri/ewpi) I can use the git repo (ewpi supports it), but I prefer a release.

I'm also ok with pre-releases (like 0.* versions)

**Comment by vitlav** [comment]
> @smohantty Beaware this, once we tag versioning, we can't change any interfaces nor behavior, we need to discuss a schedule for official versioning.

Telegram uses rlottie as external library since 1.7.13 release. Please, make a tag with some release number.

**Comment by KAMiKAZOW** [comment]
Even 0.0.1 would be fine.

**Comment by dreampiggy** [comment]
Any update to this ?

I'm a downstream dependency who build rlottie SDK integration on iOS.

Currently I have to use Git Commit Hash for dependency, which I don't think it's suitable. Because I can not ensure whether the commit I use is stable enough to use in iOS App production.

Hopeful a official release soon, at least v0.0.1 is fine.

**Comment by smohantty** [comment]
@all,
Created an inital release fro rlottie library v0.1

**Comment by dreampiggy** [comment]
I'll update the CocoaPods && Carthage && SwiftPM port of rlottie immediatelly.

For iOS user, please check: https://github.com/SDWebImage/librlottie-Xcode

**Comment by vitlav** [comment]
> @ALL,
> Created an inital release fro rlottie library v0.1
Please fix version in CMakeLists.txt, there is still
`project( rlottie VERSION 0.0.1`

**Comment by vtorri** [comment]
drop cmake instead :p

**Comment by smohantty** [comment]
corrected the version number in camke and meson also updated the release tag.

**Comment by smohantty** [comment]
> drop cmake instead :p

There are few projects who are using rlottie and using cmake build system unfortunately.

**Comment by vtorri** [comment]
if they are users, they don't care how rlottie is built. but ok, it's just that i have managed 2 build systems in the efl and it has always been a pain to deal with.

**Comment by vitlav** [comment]
> corrected the version number in camke and meson also updated the release tag.
Hey, you can't update release tag. Every change it must be a new tag.

**Comment by juippis** [comment]
I understand the fix for 0.1, and luckily it was done ~fast. Currently only Gentoo and Mageia seem to have updated to 0.1, and 0.1-1 was pushed at least to Gentoo with correct version.
https://repology.org/project/rlottie/versions

But for future yes, please make a new .1 release for small fixes.

And lastly thanks for the release!

**Comment by smohantty** [comment]
It was a mistake .. I thought no one is using it yet.
will keep in mind in future updates
